worldengine/drawing_functions.py

```diff
@@ -705,16 +691,9 @@ def on_border(pos):
 
     if verbose:
         start_time = time.time()
-    min_elev = None
-    max_elev = None
-    for y in range(world.height):
-        for x in range(world.width):
-            e = world.elevation['data'][y, x]
-            if min_elev is None or e < min_elev:
-                min_elev = e
-            if max_elev is None or e > max_elev:
-                max_elev = e
-    # elev_delta = max_elev - min_elev  # TODO: no longer used?
+    #min_elev = world.elevation['data'].min()
```

Comment:
It was there before, I didn't know if I could remove it. (Before the loop was still run, even though the variables weren't needed anywhere. So it looked less dead than it was.)